src/output-lua.c

```diff
@@ -823,8 +825,8 @@ static OutputInitResult OutputLuaLogInit(ConfNode *conf)
         } else if (opts.alproto == ALPROTO_SSH) {
             om->TxLogFunc = LuaTxLogger;
             om->alproto = ALPROTO_SSH;
-            om->tc_log_progress = SshStateBannerDone;
-            om->ts_log_progress = SshStateBannerDone;
+            om->tc_log_progress = SshStateFinished;
+            om->ts_log_progress = SshStateFinished;
```

Comment:
This can still be improved cf `OutputModule::TxLogCondition` from https://github.com/OISF/suricata/pull/5048#discussion_r439241253

Reply:
yes, I'm testing now.